chapters/7-publication.tex

```diff
@@ -432,6 +441,13 @@ \section{Preparing a complete reproducibility package}
 Major journals now often require that you provide both the data and code required to recreate your results.
 Some even require being able to reproduce the results themselves
 before they will approve a paper for publication.\sidenote{\citet{vilhuber2020report}}
+If you are producing a policy output,
+such as an open policy analysis\sidenote{\citet{hoces2018framework}}
+or some other type of material,\sidenote{Such as \url{https://github.com/worldbank/sdgatlas2018} or
+  \url{https://blogs.worldbank.org/opendata/how-we-mass-produced-reproducible-human-capital-project-country-briefs}}
```

Comment:
This is also a good example of a dissemination piece with a reproducibility package: https://blogs.worldbank.org/impactevaluations/what-development-economists-talk-about-when-they-talk-about-reproducibility